### PR TITLE
Fix formatting commands/configuration and fix Prettier/ESLint issues

### PR DESCRIPTION
**Pull request by queengooborg**
This PR performs a number of fixes relating to linting and formatting to ensure that it is running properly and catching errors.  Apparently, ESLint was never even running, and Prettier was only formatting the root files.  This PR corrects the configuration and commands, and then performs all the fixes to eliminate ESLint errors.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,16 +1,11 @@
 {
   "extends": ["eslint:recommended", "plugin:import/errors", "prettier"],
-  "rules": {
-    "no-console": [
-      "error",
-      {
-        "allow": ["info", "error"]
-      }
-    ]
-  },
```
Comment (bsmth) [inline .eslintrc.json:10]: Why is this removed?

**Review by bsmth — APPROVED** [review]
Most of the formatting changes look good to me, but @caugner should take a look about the eslint config changes, no thoughts on that, personally 👍🏻

File: .eslintrc.json (hunk 1 above)
Comment (bsmth) [reply .eslintrc.json:10]: I see it's moved into it's own config file, right?

Comment (queengooborg) [reply .eslintrc.json:10]: That's correct -- most of the files aside from that one folder SHOULD have additional console output (for example, the main MDN-BOB.js script) because they are to be run in the command line, not the browser.

**Review by caugner — COMMENTED** [review]
LGTM, but why are the file `editor/js/lib/webcomponents-bundle.js` and the dependency `@webcomponents/webcomponentsjs` being removed?

**Comment by queengooborg** [comment]
Neither are being used anywhere in the code (in fact, the first file is just an old copy of the package), so ESLint was complaining about the useless import.  I double-checked the built files and found that removing the dependency does not impact the output; all interactive examples worked as expected!

**Review by caugner — APPROVED** [review]
Awesome work! 🙏
